packages/@aws-cdk/aws-appmesh/lib/query-parameter-match.ts

```diff
@@ -0,0 +1,54 @@
+import { CfnRoute } from './appmesh.generated';
+
+// keep this import separate from other imports to reduce chance for merge conflicts with v2-main
+// eslint-disable-next-line no-duplicate-imports, import/order
+import { Construct } from '@aws-cdk/core';
+
+/**
+ * Configuration for `QueryParameterMatch`
+ */
+export interface QueryParameterMatchConfig {
+  /**
+   * Route CFN configuration for route query parameter match.
+   */
+  readonly queryParameter: CfnRoute.QueryParameterProperty;
```

Comment:
I feel like this should be called `queryParameterMatch`, to align it with other `XyzMatchConfig` interfaces.

Reply:
I agree. Thanks for point this out!